### DISCUSSION TITLE
Ability to Edit links?

### DISCUSSION THREAD

**Discussion by rami-almofleh** [sibiraj-s/ngx-editor#577] [Q&A]
Hello,

I have been using the NGX-Editor for my Angular project and have found it incredibly useful. However, one limitation I encountered is the inability to edit existing hyperlinks in the editor. As far as I can tell, there doesn't seem to be any built-in functionality for this purpose.

### Questions

1. Are there any plans to introduce the ability to edit links in future versions of NGX-Editor?
2. Is there currently any way to configure or extend the editor to enable this feature that I might have overlooked?

### Importance

The ability to edit hyperlinks is crucial for content creation and content management workflows. It would significantly enhance the editor's capabilities and usability.

### Additional Details

- **Editor Version**: 17
- **Angular Version**: 17

Thank you for your time and consideration. I look forward to your response!

Best regards

**Comment by sibiraj-s** [comment]
Yeah. Currently its not supported. You'll need to remove and the re-add the links.  I do understand this might make some things difficult while working with. But I don't have enough bandwidth to work on it actively.

Pr's to fix this is welcome.

**Reply by rami-almofleh** [reply]
Sure. I have already cloned the project and want to start the editor locally to make some changes or add the edit function to it. However, I did not understand how to start it locally. Can you help me with that, please?
